app/BUILD.bazel

```diff
@@ -594,6 +594,7 @@ android_library(
         artifact("androidx.lifecycle:lifecycle-livedata-core:jar:2.0.0-alpha1"),
         artifact("androidx.lifecycle:lifecycle-livedata-ktx"),
         artifact("com.chaos.view.pinview:1.4.4"),
+        artifact("com.google.android:flexbox:2.0.1"),
```

Comment:
This section is for binding adapters. I think we should mention this artifact in the app section. 
`name = "app",`

Reply:
Thanks, @anandwana001

Reply:
Done